x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -54,8 +79,12 @@ const FieldColumnContent = LogEntryColumnContent.extend.attrs<{
   isHighlighted: boolean;
   isHovered: boolean;
   isWrapped?: boolean;
+  isEnzyme?: boolean;
```

Comment:
The problem is that the theme prop is supposed to apply to the internal `FieldColumnContent` component, and not the exported `LogEntryFieldColumn`. Maybe if I just use the `ThemeProvider` component in the test this can work. I'll give it a shot.

Reply:
Because `ThemeProvider` uses context there's a special way I had to wrap it for Enzyme. The tests pass but the typedefs don't like what I did, seems like they're out of date relative to the Enzyme API. I'll investigate.